content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
+
+================
+Loans management
+================
+
+Odoo lets you manage your loans to have a comprehensive list of all loans undertaken by your company
+in order to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast).
+Set up your amortization schedule—or import it—and let Odoo automatically handle monthly interest
+and principal adjustments such that your financial reports are always accurate with minimal effort.
+
+Create a new loan
+-----------------
+
+You can create a new loan by going to :menuselection:`Accounting --> Accounting --> Loans`. When
+creating a new loan, you can create your amortization schedule in three ways:
+
+- By importing it from a supported file;
+- By calculating it from a couple of values (e.g. the :guilabel:`Amount Borrowed`, the
+  :guilabel:`Duration`, etc.);
+- By manually filling in the lines of the schedule.
+
+In either case, 3 different fields are required for each line of the amortization schedule: the
+:guilabel:`Date`, the :guilabel:`Principal` and the :guilabel:`Interest`.
+
+The :guilabel:`Amount Borrowed`, :guilabel:`Interest` and :guilabel:`Duration` fields will be red
+if the sum of the lines does not match the totals amounts you have entered.
+
+Loan entries mechanism
+----------------------
+
+When the amount borrowed is credited to your bank account, it should be transferred to a long-term
+account (defined in the loan settings). Then, upon the validation of the loan, Odoo will create all
+the necessary journal entries such that you can always have a holistic and forecast view of upcoming
+due dates. The entire process is completely automated with a reclassification of long-term and
+short-term principal reclassification mechanism.
```

Comment:
```suggestion
When the amount borrowed is credited to a bank account, it should be transferred to a long-term
account (defined in the :guilabel:`Loan Settings` tab). Then, upon the validation of the loan, Odoo
creates the necessary journal entries so that there is always a holistic and forecasted view of
upcoming due dates. The entire process is completely automated with a long-term and short-term
principal reclassification mechanisms.
```
Added a gui label and rephrased a few things. Can you double check that the last line is clear and accurate? You had "reclassification" in there twice which seemed wrong to me but I want to be sure.

Reply:
Woops indeed, it's better like that